codelists/+itemClassificationScheme.csv

```diff
@@ -1,2 +1,3 @@
 Code,Title,Description,Source,Category
 CELEX,CELEX number,Identifier scheme for EUR-Lex documents,https://eur-lex.europa.eu/content/help/faq/intro.html#help8,legalBasis
+LexML,LexML URN,Identifier scheme for BR-Lex documents,https://projeto.lexml.gov.br/,legalBasis
```

Comment:
Ah, sorry, I forgot we had some style suggestions for codes, so that they are more predictable for implementers and users. In that case, we can leave the code uppercase, but use the regular capitalization in the title.

```suggestion
LEXML,LexML URN,Identifier scheme for BR-Lex documents,https://projeto.lexml.gov.br/,legalBasis
```